src/views/transaction-details/components/chain-row/chain-row.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from "react";
+
+import { ReactComponent as EthChainIcon } from "src/assets/icons/ethereum.svg";
+import { ReactComponent as HermezChainIcon } from "src/assets/icons/polygon-hermez.svg";
+import useChainRowStyles from "src/views/transaction-details/components/chain-row/chain-row.style";
+
+interface ChainRowProps {
+  type: "l1" | "l2";
+  side: "to" | "from";
+}
+
+const ChainRow: FC<ChainRowProps> = ({ type, side }) => {
+  const classes = useChainRowStyles();
+
+  const chain = {
+    l1: { from: "hermez", to: "eth" },
+    l2: { from: "eth", to: "hermez" },
+  };
+
+  switch (chain[type][side]) {
+    case "eth":
+      return (
+        <>
+          <EthChainIcon /> Ethereum chain
```

Comment:
If we are going to return an icon with text, I would use proper HTML elements:

```
return (
  <div>
    <EthChainIcon />
    <Typography type="body">Ethereum chain</Typography>
  </div>
)

Reply:
It's wrapped by `Typography`in `transaction-details.view`

Reply:
Then the wrapping should be done here

Reply:
An icon wrapped by typography?